htdocs/xoops_lib/Xoops.php

```diff
@@ -927,7 +927,7 @@ public function getHandlerUser($optional = false)
      * @param string  $name     name of handler
      * @param boolean $optional true if failure to load handler should be considered a warning, not an error
      *
-     * @return XoopsObjectHandler|XoopsPersistableObjectHandler|null
+     * @return \Xoops\Core\Kernel\XoopsPersistableObjectHandler|XoopsObjectHandler|XoopsPersistableObjectHandler|null
```

Comment:
Scrutinizer did not like these much:

https://scrutinizer-ci.com/g/XOOPS/XoopsCore/inspections/c5e07acd-f5e4-4406-8512-59bc4d4b9c62/issues/

I tend to rely on its observations. Sometimes they are useless, sometimes they catch some hard to notice errors.